lib/libvmod_vtc/vmod_vtc.vcc

```diff
@@ -169,6 +169,20 @@ version ``v1``).
 Candidate for moving into vmod_proxy, but there were concerns about
 the interface design
 
+$Function DURATION benchmark(SUB, INT total=1000, INT leg=10)
+
+Benchmark a VCL sub and return the average runtime.
+
```

Comment:
Is this generally useful enough that it ought to live in `vmod_std`, or is it too dangerous for that ?

Reply:
I would think benchmarking was a test function and nothing for production, thus the `vtc` vmod